specification/mediaservices/resource-manager/Microsoft.Media/Accounts/stable/2023-01-01/Accounts.json

```diff
@@ -509,6 +509,39 @@
           "description": "The Private Endpoint Connections created for the Media Service account.",
           "type": "array",
           "readOnly": true
+        },
+        "minimumTlsVersion": {
+          "type": "string",
+          "enum": [
+            "Tls10",
+            "Tls11",
+            "Tls12",
+            "Tls13"
+          ],
+          "x-ms-enum": {
+            "name": "MinimumTlsVersion",
+            "values": [
+              {
+                "value": "Tls10",
+                "description": "Minimum TLS version is TLS 1.0."
+              },
+              {
+                "value": "Tls11",
+                "description": "Minimum TLS version is TLS 1.1."
+              },
```

Comment:
Do you really want to allow people to use TLS 1.0 and 1.1? You are probably going to get dinged for it, if you do, since they're obsolete.

Reply:
There are some customers who are using TLS 1.0 and 1.1, and it is only those customers who can use these lower TLS versions. We have a check in our service that returns an error if any other customer tries to use these values.